source/jormungandr/tests/bss_provider_tests.py

```diff
@@ -367,3 +366,47 @@ def pois_without_stands_on_places_autocompletion_test(self):
             poi = places[0]['poi']
             is_valid_poi(poi)
             assert not 'stands' in poi
+
+    ############################
+    #####
+    ##### API: journeys
+    ##### BSS support: Yes
+    #####
+    ############################
+
+    def test_bss_stands_on_journeys_disabled_by_default(self):
+        supported_pois = ['station_1', 'station_2']
+        with mock_bss_providers(pois_supported=supported_pois):
+            query = journey_basic_query + "&first_section_mode=bss&last_section_mode=bss"
+            response = self.query_region(query)
+            self.is_valid_journey_response(response, query)
+
+            journeys = get_not_null(response, 'journeys')
+
+            for place in self._get_from_to_places_of_journeys_sections(journeys):
+                if 'poi' in place:
+                    assert 'stands' not in place['poi']
+
+    def test_journey_sections_from_to_poi_with_stands(self):
+        supported_pois = ['station_1']
+        with mock_bss_providers(pois_supported=supported_pois):
+            query = journey_basic_query + "&first_section_mode=bss&last_section_mode=bss&bss_stands=true"
+            response = self.query_region(query)
+            self.is_valid_journey_response(response, query)
+
+            journeys = get_not_null(response, 'journeys')
+            for place in self._get_from_to_places_of_journeys_sections(journeys):
+                if 'poi' in place:
+                    poi = place['poi']
+                    is_valid_poi(place['poi'])
+                    if place['poi']['id'] in supported_pois:
+                        assert 'stands' in poi
+                        assert poi['stands']['available_places'] == 13
+                        assert poi['stands']['available_bikes'] == 3
+                        assert poi['stands']['total_stands'] == 16
+                    else:
+                        assert 'stands' not in poi
+
+    def _get_from_to_places_of_journeys_sections(self, journeys):
+        # utility that returns 'from' and 'to' places for each section of each given journey
+        return (s[from_to] for j in journeys for s in j['sections'] for from_to in ('from', 'to') if from_to in s)
```

Comment:
`for place in (s.get(from_to, {}) for j in journeys for s in j.get('sections', []) for from_to in ('from', 'to'))`